scripts/preprod-network.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Please refer to the 'GETTING_STARTED.md' guide for more details.
+
+set -e
+echo "
+    ###########################################################
+    #                   !!! WARNING !!!                       #
+    # Do not use this script for production deployments.      #
+    # This script provides a quick way to start the services  #
+    # needed for learning and exploring the Cardano SDK.      #
+    # It does not implement any security policies, UAC or     #
+    # other topics related to production deployments.         #
+    ###########################################################
+"
+
+export NETWORK="preprod"
```

Comment:
Seems like this script could easily support any network
```suggestion
export NETWORK=$n
```

Reply:
Reasonable. Let's keep it as is for now